gazebo_ros/launch/gzclient.launch.py

```diff
@@ -54,23 +69,28 @@ def generate_launch_description():
                               description='Set "true" to run gzserver with gdb'),
         DeclareLaunchArgument('valgrind', default_value='false',
                               description='Set "true" to run gzserver with valgrind'),
+        DeclareLaunchArgument('gui_required', default_value='false',
```

Comment:
it might be a bit late but `gui_required` inspire my something like "we want to spawn the GUI for sure when running the roslaunch". The description is far away from this thought and explains the shutdown behavior.

Reply:
Yeah... that makes sense. Do you have a suggestion in mind? It's kind of an odd thing to name. I used *_required following the ROS 1 syntax of `<node required>`, assuming somebody who wants this option would know what that is. But to somebody who doesn't, it is more like the meaning you said.

If we want the flags to be the same as the ROS 1 ones, we should change the names in Kinetic, Melodic, Noetic too, since they have just been added and not released (I think?) yet.

Reply:
So... we released melodic today, so I think it's too late to change the parameter :confused:

Reply:
let's live with it, no big problem. The help is clear and we can document the parameters in the tutorials to mitigate problems with naming.